### DISCUSSION TITLE
Real name in user profile?

### DISCUSSION THREAD

**Discussion by ykundesu** [community/community#69358] [GitHub Education]
### Hi everyone,

Hi, I submitted an application.
Then it came back that I need to write my real name and background in my user profile. This is a risky practice and I do not want to do it. It also says I need to capture your picture. Is this a student book? Or is it your face? Or both?
Full text:
Have you completed your GitHub user profile with all your relevant information, such as your full name as it appears in your image and a short bio?

If your school does not provide academic email then you must enable use of your device camera to capture your image.

**Comment by mantasu** [comment]
From my experience, at the very least you should write your first name on your profile, besides you can change what is displayed on your profile later after approval. For the camera picture, it should be your student ID with the expiry date.
